### PR TITLE
Fix zmq lost messages

### PR DESCRIPTION
**Pull request by fanatid**
I'm not sure, but probably this fix issue related with lost messages. (I did not run service)

### CODE REVIEW

File: service.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ let debugZmqTx = require('debug')('service:zmq:tx')
 let Indexd = require('indexd')
 let leveldown = require('leveldown')
 let rpc = require('./rpc')
-let zmq = require('zmq')
+let zmq = require('zeromq')
```
Comment (dcousens) [inline service.js]: why change this?

File: service.js [hunk 2]
```diff
@@ -26,18 +26,19 @@ module.exports = function initialize (callback) {
     zmqSock.subscribe('hashblock')
     zmqSock.subscribe('hashtx')
 
-    let lastSequence = 0
+    let sequences = {}
```
Comment (dcousens) [inline service.js]: oh,  I wonder if that is the issue.

File: service.js (hunk 1 above)
Comment (dcousens) [reply service.js]: (I mean,  seams like a good change,  but,  why?)

Comment (fanatid) [reply service.js]: check `zmq` repo page: https://github.com/JustinTulloss/zeromq.node
project is moved to `zeromq`

File: service.js (hunk 2 above)
Comment (fanatid) [reply service.js]: well, I logged messages to console and sequence for block and tx was different

**Comment by dcousens** [comment]
Thanks @fanatid :heart:

**Pull request merged**